### DISCUSSION TITLE
How did I get a newer version than the current release? v1.1.12986

### DISCUSSION THREAD

**Discussion by datatimp** [microsoft/winget-cli#1741] [Q&A]
`winget --info` 
```
Windows Package Manager v1.1.12986
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19041.1348
Package: Microsoft.DesktopAppInstaller v1.16.12986.0
```

I am not on the Windows Insider program. I installed manually. So, I'm not sure how I am on **v1.1.12986**. Any ideas?

**Comment by ccnly** [comment]
Same question.  A fresh installation of Windows shows the version as `v1.2.3131-preview`.
